### PR TITLE
Fix Az.Monitor XML Comments

### PR DESCRIPTION
**Pull request by CaptainFanZzz**
## Description

Fix Az.Monitor XML Comments

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Monitor/Monitor/OutputClasses/PSVoiceReceiver.cs [hunk 1]
```diff
@@ -25,7 +25,7 @@ public class PSVoiceReceiver : PSActionGroupReceiverBase
         // <summary>Gets or sets the country code.</summary>
```
Comment (v-yuzhichen) [inline src/Monitor/Monitor/OutputClasses/PSVoiceReceiver.cs:25]: I think this should be "/// <summary>Gets or sets the country code.</summary>"

File: src/Monitor/Monitor/Monitor.csproj [hunk 2]
```diff
@@ -8,6 +8,15 @@
 
   <PropertyGroup>
     <RootNamespace>$(LegacyAssemblyPrefix)Insights</RootNamespace>
+    <GenerateDocumentationFile>True</GenerateDocumentationFile>
```
Comment (v-yuzhichen) [inline src/Monitor/Monitor/Monitor.csproj]: Please do not check in the changes in csproj file

**Review by v-yuzhichen — APPROVED** [review]

File: src/Monitor/Monitor/OutputClasses/PSDynamicMetricCriteria.cs [hunk 3]
```diff
@@ -29,7 +29,7 @@ public class PSDynamicMetricCriteria : DynamicMetricCriteria, IPSMultiMetricCrit
         /// <summary>
         /// Initiliazes a PS object for dynamic metric criteria
         /// </summary>
-        /// <param name="metricCriteria">The original dynamic metric criteria object</param>
+        /// <param name="dynamicMetricCriteria"></param>
```
Comment (BethanyZhou) [inline src/Monitor/Monitor/OutputClasses/PSDynamicMetricCriteria.cs:32]:
```suggestion
        /// <param name="dynamicMetricCriteria">The original dynamic metric criteria object</param>
```

File: src/Monitor/Monitor/OutputClasses/PSMetricDimension.cs [hunk 4]
```diff
@@ -22,9 +22,9 @@ public class PSMetricDimension
         /// <summary>
         /// Initializes a new instance of the MetricDimension class.
         /// </summary>
-        /// <param name="dimension">Name of the dimension.</param>
-        /// <param name="includeValues">List dimension includeValues.</param>
-        /// <param name="excludeValues">List of dimension excludeValues.</param>
+        /// <param name="dimensionName"></param>
+        /// <param name="valuesToInclude"></param>
+        /// <param name="valuesToExclude"></param>
```
Comment (BethanyZhou) [inline src/Monitor/Monitor/OutputClasses/PSMetricDimension.cs:27]:
```suggestion
        /// <param name="dimensionName">Name of the dimension.</param>
        /// <param name="valuesToInclude">List dimension includeValues.</param>
        /// <param name="valuesToExclude">List of dimension excludeValues.</param>
```

File: src/Monitor/Monitor/TransitionalClasses/MetricSettings.cs [hunk 5]
```diff
@@ -35,7 +35,7 @@ public class MetricSettings : Monitor.Models.MetricSettings
         /// <summary>
         /// Initializes a new instance of the MetricSettings class.
         /// </summary>
-        /// <param name="logSettings">The metric settings</param>
+        /// <param name="metricSettings"></param>
```
Comment (BethanyZhou) [inline src/Monitor/Monitor/TransitionalClasses/MetricSettings.cs:38]:
```suggestion
        /// <param name="metricSettings">The metric settings</param>
```

**Review by BethanyZhou — COMMENTED** [review]
Please try your best to utilize existing information rather than deleting them directly.

**Review by BethanyZhou — CHANGES_REQUESTED** [review]
Please have a look

**Review by BethanyZhou — APPROVED** [review]

**Pull request merged**